src/checkers/human_checker.rs

```diff
@@ -46,6 +43,16 @@ pub fn human_checker(input: &CheckResult) -> bool {
     cli_pretty_printing::success(&format!("DEBUG: Human checker returning: {}", result));
 
     if !result {
+        let fd_result = database::insert_human_rejection(&input.text, input);
+        match fd_result {
+            Ok(_) => (),
+            Err(e) => {
+                cli_pretty_printing::warning(&format!(
+                    "DEBUG: Failed to log human checker rejection due to error: {}",
```

Comment:
```suggestion
                    "DEBUG: Failed to write human checker rejection due to error: {}",
```

we are doing a human checker write here, so i think this word is better?

Reply:
Yeah, that makes sense to me.